src/utils/listenerHelpers.ts

```diff
@@ -0,0 +1,91 @@
+import { DispatchedEvent } from "globals/Calc";
+import { Calc, Console } from "globals/window";
+
+const dispatchOverridingHandlers: [
+  (evt: DispatchedEvent) => boolean | undefined,
+  number,
+  number
+][] = [];
```

Comment:
Triple is unclear. Would be better to be an interface type instead of tuple type, so it's clear what is `.priority` vs `.id`.

Reply:
Fixed